### ISSUE TITLE
Does not support https:// repos

### ISSUE THREAD

**Issue by cclauss** [CircleCI-Public/circleci-config-generator#5]
$ __git remote -v__
```
origin    https://github.com/me/repo (fetch)
origin    https://github.com/me/repo (push)
upstream  https://github.com/them/repo (fetch)
upstream  https://github.com/them/repo (push)
```

Fails (on Mac OSX) with...
```
$ ./cci-config-generator.sh
Gathering info please wait...
fatal: 'git@com/me/repo (fetch).com:.git' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Paste your CircleCI API token here:
```

**Comment by eddiewebb** [comment]
@Br3nda , @cclauss  - would you please try the referenced https://github.com/CircleCI-Public/circleci-config-generator/blob/issue-5-issue-1/cci-config-generator.sh which should address this issue?

**Comment by eddiewebb** [comment]
The previous error was unrelated to OS< and assumed the presence of   @,:,. as field delimiters.  the @ will not be present in HTTPS formatted URLs.

I updated to use `git remote get-url --push origin | perl -ne 'print $1 if /([^\/:]*\/[^\/]*?)(.git)?$/'` which is passing the patterns I gleamed from all open issues

**Comment by eddiewebb** [comment]
Fixed in #9
